mmedit/models/editors/nafnet/local_utils.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
```

Comment:
Combining `utils.py` and `local_utils.py` as `nafnet_module.py` would be better.